### DISCUSSION TITLE
Reduce PDF file size generated using domPdf

### DISCUSSION THREAD

**Discussion by maya1987** [dompdf/dompdf#3073] [General]
Hi
I reduced the pdf file size using configuration 'enable_font_subsetting ' = true .The size was reduced from 40kb to 22 kb. But I want to reduce it to around 10kb. The original pdf size is 10kb .How is this possible?

Regards,
Maya A

**Comment by bsweeney** [comment]
That really depends on the nature of the content of the PDF. If you have images, you should optimize them prior to rendering the PDF since Dompdf does not currently perform image optimization. You could also try lowering the Dompdf DPI, which can impact how some images are rendered.
